app/assets/stylesheets/_buttons.scss

```diff
@@ -69,6 +68,46 @@ a .button,
 .orange_submit_button { padding: 5px; }
 .translation_button { @extend .play_video_button; }
 
+.buttons-container {
+  bottom: 0;
+  display: flex;
+  justify-content: space-between;
+  left: 0;
+  text-align: left;
+  width: 100%;
+  .add-to-vocab-btn {
+    display: flex;
+  }
+}
+
+.download-link {
+  width: 50%;
```

Comment:
could this be changed to flex:50% please so it stays consistent with the other flex properties

Reply:
@Rabid-Dan Done!

Reply:
@louise-r-blue I'm not seeing the change? It still displays as width 50%